promtimer/annotations.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020 Couchbase, Inc All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from os import path
+import time
```

Comment:
unnecessary import

Reply:
`time` is not used, but `path` is on line 160:
```
if not path.isfile(FILENAME):
```
Is it unnecessary because it's assumed to be present from `promtimer.py`? (and if so, is that not an assumption that should be made since `annotations.py` should be self-sufficient?)